### PR TITLE
Rename country_{,code} parameter for consistency

### PR DESCRIPTION
**Pull request by mfilej**

### CODE REVIEW

**Comment by koleksiuk** [comment]
:star:

**Comment by mfilej** [comment]
Waiting before merging because I'm not sure if we're relying on this contract somewhere.

File: lib/gemonames/api_client.rb [hunk 1]
```diff
@@ -51,15 +51,15 @@ def reverse_find(**args)
       )
     end
 
-    def countries_info(country: nil)
+    def countries_info(country_code: nil)
```
Comment (mfilej) [inline lib/gemonames/api_client.rb:54]: @RadekMolenda does this parameter make sense (we should be able to skip it I think)?

**Comment by RadekMolenda** [comment]
:+1:

**Pull request merged**